lib/react-query-config.ts

```diff
@@ -0,0 +1,9 @@
+export const config = {
+    defaultOptions: {
+        queries: {
+            staleTime: 1 * 60 * 60 * 1000,
+            cacheTime: 5 * 60 * 60 * 1000,
+            refetchOnWindowFocus: false
+        }
+    }
```

Comment:
Great to see that we are caching the responses and not spamming the APIs. Thinking on your feet!